nms/app/packages/magmalte/e2e_test_setup.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+#
+# Copyright 2020 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+mkdir -p .cache
+mkdir -p /tmp/nms_artifacts
+openssl req -nodes -new -x509 -batch -keyout .cache/mock_server.key -out .cache/mock_server.cert
+docker-compose --env-file .env.mock -f docker-compose-e2e.yml up -d
+sleep 30
```

Comment:
You might want to do something more sophisticated here since especially in CI the duration of operations can sometimes take a lot longer than usual. Anything we can do to decrease flakiness is valuable here.

Reply:
Will do this in future PRs